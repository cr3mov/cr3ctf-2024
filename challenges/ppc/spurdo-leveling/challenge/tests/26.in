104238068215926781 947729975817044212
1 3432361759460920
1 41071986872490480
