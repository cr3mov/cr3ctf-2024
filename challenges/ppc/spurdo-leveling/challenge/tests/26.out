19 19 19
